dev/io.openliberty.microprofile.health.4.0.internal_fat/publish/servers/MPHealth40And22/jvm.options

```diff
@@ -0,0 +1 @@
+-Dcom.ibm.ws.beta.edition=true
```

Comment:
I don't believe this file and property are required since you don't have any beta guarding with this change.  Features being marked beta is about packaging of shipped features and should not impact things for running tests.